src/test/java/seedu/foodrem/logic/commands/itemcommands/DeleteCommandTest.java

```diff
@@ -23,6 +23,7 @@
  * {@code DeleteCommand}.
  */
 public class DeleteCommandTest {
+    private static final String EXPECTED_FORMAT_SUCCESS = "Deleted Item:\n%1$s";
```

Comment:
Hm, I guess making the SUCCESS_MESSAGES private would cause issues when testing. I'm thinking we can possibly abstract these messages to some Class that handles messages, which might follow OCP (because if we add additional messages in the future, we don't modify the Command classes). Perhaps by abstracting these messages to some class that would handle these messages depending on the Command, it might also follow SRP (We only change this new Class that handles MESSAGES). What do you think? Perhaps this might improve testability, considering that MESSAGES are private.

Reply:
See my previous comment on why I made this private, and instead, made total two copies of this format string, both are private, one in the test class, and one in the SUT class.

Reply:
Specifically, [this comment](https://github.com/AY2223S1-CS2103T-W16-2/tp/pull/222#discussion_r996732082).